examples/flutter_gallery/web/index.html

```diff
@@ -3,10 +3,21 @@
 Use of this source code is governed by a BSD-style license that can be
 found in the LICENSE file. -->
 <html>
-    <head>
-        <title>Flutter Gallery</title>
-    </head>
-    <body>
-        <script src="main.dart.js"></script>
-    </body>
+<head>
+  <meta charset="UTF-8">
+  <meta content="IE=Edge" http-equiv="X-UA-Compatible">
```

Comment:
by convention the http-equiv really should come first, it's the key and content is the value.